src/redux/products/tests/products.reducer.test.js

```diff
@@ -0,0 +1,201 @@
+import productReducer, { initialState } from '../products.reducer';
+import {
+  setAllFilterProducts,
+  setCategoryFilter,
+  setColorsFilter,
+  setPatternsFilter,
+  setPriceFilter,
+  setLoading,
+  setSearchFilter,
+  setHotItemFilter,
+  setPagesCount,
+  setCurrentPage,
+  setSortByPrice,
+  setSortByRate,
+  setSortByPopularity
+} from '../products.actions';
+
+describe('Product reducer test', () => {
+  const productsExample = [
+    {
+      name: [
```

Comment:
I think it is better to create some helper file with product's mock data to use in tests for reducers, sagas and actions to reduce amount of code

Reply:
Done